codespell_lib/data/dictionary.txt

```diff
@@ -20195,6 +20215,10 @@ reprocuce->reproduce, reprocure,
 reprocuced->reproduced, reprocured,
 reprocuces->reproduces, reprocures,
 reprocucing->reproducing, reprocuring,
+reprodice->reproduce
+reprodiced->reproduced
+reprodicing->reproducing
+reprodiction->reproduction
```

Comment:
Can you also add the dic->duc variants of 
```
reproducibly
reproducibility
reproducible
reproducibility
reproducibly
reproducibility
```

Reply:
I added 
```
reprodicible->reproducible
reprodicibly->reproducibly
```

Reply:
Thanks, any reason for not adding the others? If dic->duc is a valid typo, the others are likely to occur sometimes too.

Reply:
Sorry, thought i mentioned it but the list above has repetitive entries. I just added the ones that were unique. Also forgot to mention I added `reprodicibility->reproducibility`

Reply:
Apologies, so it does!